indico/modules/events/contributions/templates/display/contribution_display.html

```diff
@@ -259,7 +259,13 @@ <h3>{% trans %}Subcontributions{% endtrans %}</h3>
     {% if contribution.event.has_feature('papers') %}
         {{ render_paper_section(contribution) }}
     {% endif %}
+    {% if contribution.event.has_feature('editing') %}
+        {{ render_editables_section(contribution) }}
+    {% endif %}
     <script>
+        if (document.querySelector('#editable-submission-button')) {
```

Comment:
That's a bit ugly. I thought about checking (in the jinja/pyhjon code) whether a new revision can be submitted but it'd result in duplicating logic, so I'd move the check inside the function then (just return early if you don't have the button).

Also, no need to call this if the editing feature isn't enabled. You can just use a jinja if to not render the function call at all in this case.